server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -595,6 +596,17 @@ public Iterator<Setting<?>> settings() {
         Property.ServerlessPublic
     );
 
+    private static final FeatureFlag LOGS_INDEX_MODE_ENABLED_FEATURE_FLAG = new FeatureFlag("logs_index_mode_enabled");
+
+    /**
+     * if index.mode "logs" is applied by default in logs@settings for 'logs-*-*'
+     */
+    public static final Setting<Boolean> LOGS_INDEX_MODE_ENABLED = Setting.boolSetting(
```

Comment:
This isn't really an index setting, but a node setting. Maybe rename it with the `index.` prefix (`cluster.logsdb.enabled`?) and to another file?

Reply:
is a setting for "index.mode"...